src/main/java/com/xpanxion/java/springboot/da1/demo/service/student7/DataService7.java

```diff
@@ -0,0 +1,115 @@
+package com.xpanxion.java.springboot.da1.demo.service.student7;
+
+import com.xpanxion.java.springboot.da1.demo.model.student7.Book;
+import com.xpanxion.java.springboot.da1.demo.model.student7.BookstoreBook;
+import com.xpanxion.java.springboot.da1.demo.model.student7.Manager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Service
+public class DataService7 {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    @Autowired
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+    private final String SELECT_MANAGERS = "select * from manager";
+    private final String INSERT_MANAGER = "insert into student7_da.manager(full_name) Values (?)";
+    private final String UPDATE_MANAGER = "UPDATE manager SET full_name= ? WHERE manager_id= ?";
+    private final String SELECT_BOOK = "select * from book where book_id = ?";
+    private final String INSERT_BOOK = "INSERT INTO book(title, isbn, price) Values (:title, :isbn, :price)";
+    private final String LINK_BOOKSTORE = "INSERT INTO bookstore_book(bookstore_id, book_id,quantity) VALUES (:bookstoreId,:bookId,:quantity)";
+    private final String SELECT_BOOKSTORE_BOOK = """
+            "SELECT bk.book_id, title, isbn, bsb.quantity, price
+            " +
```

Comment:
I just noticed that we don't need to add the plus operator (`+`) and we don't need to include the quotes (`"`).  All we need to do is this:

```
select ...
from
  join ...
  join ..
where ...
```